src/server/api/reply.ts

```diff
@@ -0,0 +1,68 @@
+import { Type } from '@sinclair/typebox'
+import type { APIConfig, FastifyTypebox } from '.'
+import type Store from '../store'
+import type ActivityPubSystem from '../apsystem.js'
+
+export const replyRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityPubSystem) => async (server: FastifyTypebox): Promise<void> => {
+  // Get list of replies for a post
+  server.get<{
+    Params: {
+      actor: string
+      postURL: string
+    }
+    Reply: string | string[]
+  }>('/:actor/replies/:postURL', {
+    schema: {
+      params: Type.Object({
+        actor: Type.String(),
+        postURL: Type.String() // This will represent the post's URL to fetch the replies for
+      }),
+      response: {
```

Comment:
Instead of a list of urls this should be an unordered collection like the inbox GET